### ISSUE TITLE
s2e init error

### ISSUE THREAD

**Issue by Owlz** [S2E/s2e-env#36]
So i've gone back and am now running the s2e tool inside 16.04.2 TLS. I mentioned the setup problems over in the docs section (probably should have opened that ticket here).

Once I got past that problem, now init is having problems:

```
s2e init /home/s2e/s2e
INFO: [init] Creating environment in /home/s2e/s2e
INFO: [init] Installing S2E dependencies
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'libsdl1.2-dev' instead of 'libsdl-dev'
E: Unable to locate package libncurses5-dev:i386
E: Unable to locate package libglib2.0-0:i386
E: Couldn't find any package by glob 'libglib2.0-0'
E: Couldn't find any package by regex 'libglib2.0-0'
ERROR: [init] 

  RAN: /usr/bin/sudo -S apt-get install build-essential cmake wget git texinfo flex bison python-dev libguestfs-tools genisoimage python-pip xz-utils docker.io p7zip-full pxz fuse libhivex-bin libdwarf-dev libelf-dev libiberty-dev binutils-dev libreadline-dev libboost-dev zlib1g-dev libjemalloc-dev nasm pkg-config libmemcached-dev libvdeplug-dev libpq-dev libc6-dev-i386 libboost-system-dev libboost-serialization-dev libboost-regex-dev libprotobuf-dev protobuf-compiler libbsd-dev libsigc++-2.0-dev libsdl-dev libglib2.0-dev python-docutils qemu mingw-w64 lcov libprocps4-dev lib32ncurses5 lib32ncurses5-dev libncurses5-dev libx32ncurses5-dev libncurses5-dev:i386 libglib2.0-0:i386

  STDOUT:


  STDERR:
```

For reference:
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.2 LTS
Release:        16.04
Codename:       xenial
```

**Comment by Owlz** [comment]
Looks like docker base ubuntu:16.04 doesn't come w/ 32-bit libraries enabled. Need to do the following

```bash
sudo dpkg --add-architecture i386
```

**Comment by adrianherrera** [comment]
Hey Owlz,

The errors are occurring because `s2e init` is trying to install 32-bit versions of libncurses5-dev and libglib2.0-0. I've fixed this in PR #37.

Thanks for noticing this!

~Adrian
